src/App.test.js

```diff
@@ -1,8 +1,16 @@
-import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from './redux/configureStore';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe('App Component Rendering', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
```

Comment:
The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly 😉 Fantastic job!